src/content/spotlights/what-is-tailwind.md

```diff
@@ -0,0 +1,386 @@
+---
+authors:
+  - "Thias#8888"
+  - "jun_ok#0001"
+  - "shan#6094"
+  - "propagandalf#0001"
+created_at: "2023/09/10"
+title: What is Tailwind?
+---
+
+Tailwind
+**What is Tailwind CSS?**
```

Comment:
Headings should be used instead of bold, [see other spotlight pages](https://raw.githubusercontent.com/the-programmers-hangout/website/main/src/content/spotlights/what-is-ansible.md).